src/overlays/actors/ovl_En_Kanban/z_en_kanban_gfx.h

```diff
@@ -0,0 +1,71 @@
+#ifndef Z_EN_KANBAN_GFX_H
```

Comment:
Why is this file a header? It should be a `.c` file since it actually contain variable definitions

Reply:
It creates a different object file and gives me a whole bunch of errors which I don't know how to fix if I make it a .c
```
cfe: Error: src/overlays/actors/ovl_En_Kanban/z_en_kanban_gfx.c, line 1: Syntax Error
 static u16 sShadowTexFlags[] = {
 -----------^
cfe: Error: src/overlays/actors/ovl_En_Kanban/z_en_kanban_gfx.c, line 1: Empty declaration specifiers
 static u16 sShadowTexFlags[] = {
 -----------^```

Reply:
You should do what OoT does here. Basically you need to `#include` the actor's header too
https://github.com/zeldaret/oot/blob/master/src/overlays/actors/ovl_En_Kanban/z_en_kanban_gfx.c

Reply:
I don't understand why including the header is required or makes it work? How does that change it? It's only a single array...

Reply:
Because the types we use (like `u16`) are not standard types, those are typedef defined in other header. `#include`ing them should fix your issue